python/nano/tutorial/inference/openvino/openvino_inference_async.py

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Required Dependecies
+
+# ```bash
+# pip install openvino-dev
+# ```
+

```

Comment:
add methods to download data and checkpoints here rather than host them in bigdl repo

Reply:
could we find some meaningful case here (not necessary)

Reply:
> could we find some meaningful case here (not necessary)
The model in this script can be used for image classification task, maybe we could use a real image as input ? Or find some other tasks?

Reply:
if that's the case, it should be OK, no need to use real image as input